### PR TITLE
Transform chapter-links to glimmer component

### PR DESCRIPTION
**Pull request by kennstenicht**
For better readability, I have only formatted the the code in the first commit. I would recommend to ignore whitespaces.

### CODE REVIEW

File: guidemaker-ember-template/src/components/chapter-links.hbs [hunk 1]
```diff
@@ -1,18 +1,39 @@
...
+    </:previous>
 
     <:next>
-      <LinkTo @route="version.show" @model={{this.page.nextPage.url}} class="next-guide">
+      <LinkTo
+        @route="version.show"
+        @model={{this.page.nextPage.url}}
+        class="next-guide"
+      >
+      {{log this.page.nextPage}}
```
Comment (mansona) [inline guidemaker-ember-template/src/components/chapter-links.hbs:22]:
```suggestion
```

**Comment by mansona** [comment]
I've merged your other PR so I guess this one needs to be rebased to continue?

**Comment by kennstenicht** [comment]
@mansona I have rebased the branch.

**Review by mansona — APPROVED** [review]
:shipit:

**Pull request merged**